test/phoenix/digester_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Phoenix.DigesterTest do
+  use ExUnit.Case, async: true
+
+  test "fails when the given paths are invalid" do
+    assert {:error, :invalid_path} = Phoenix.Digester.compile("../some invalid?/path", "/other_invalid ?? /path")
+  end
+
+  test "digests and compress files" do
+    output_path = Path.join("tmp", "digested")
```

Comment:
Also we are not testing that compiling twice won't regenerate files (that for example we won't have `.gz.gz`).